test/fixtures/charts.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
We're not going to be using fixtures in our testing procedure, so we don't need this.

Reply:
This isn't actionable though, so don't worry about it for now.